dev-install-locally-no-tests.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+./gradlew clean install -x test
```

Comment:
Sure thing

On Fri, Apr 8, 2016 at 07:18, Misagh Moayyed <notifications@github.com> wrote:
In dev-install-locally-no-tests.sh [https://github.com/Jasig/cas/pull/1676#discussion_r59010910] :

> @@ -0,0 +1,3 @@
> +#!/usr/bin/env bash
> +
> +./gradlew clean install -x test


This one for instance:
https://github.com/Jasig/cas/blob/master/dev-build-no-tests.sh [https://github.com/Jasig/cas/blob/master/dev-build-no-tests.sh]

You added these, remember? :D

I have no objections to the change, just the organization of it.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub [https://github.com/Jasig/cas/pull/1676/files/9c8a4b49aee43d91c0113ec6f61045218cf9e71f#r59010910]